content/facility.yaml

```
@@ -1,168 +1,153 @@
+    img: mayku
+    url: https://www.mayku.me/
 
-- name: Silver Bullet Die Cutter
-  img: silver-bullet
-  url: https://silverbulletcutters.com/
-
-- name: Epilog Fusion M2 40inch Laser Cutter
-  img: epilog
-  url: https://www.epiloglaser.com/laser-machines/fusion-laser-series.htm
+  - name: Silver Bullet Die Cutter
+    img: silver-bullet
+    url: https://silverbulletcutters.com/
 
+  - name: Epilog Fusion M2 40inch Laser Cutter
+    img: epilog
+    url: https://www.epiloglaser.com/laser-machines/fusion-laser-series.htm
 
 Electronics:
-- name: Voltera V-One PCB Printer
-  img: voltera
-  url: https://www.voltera.io/
-
-- name: Bantam PCB Mill
-  img: bantam
-  url: https://www.bantamtools.com/
+  - name: Voltera V-One PCB Printer
+    img: voltera
+    url: https://www.voltera.io/
 
-- name: Weller WE1010NA
-  img: wellner
-  url: https://www.weller-tools.com/we1010na/
+  - name: Bantam PCB Mill
+    img: bantam
+    url: https://www.bantamtools.com/
 
-- name: Eventek KPS305D DC Power Supply
-  img: eventek
-  url: https://www.amazon.com/dp/B071RNT1CD
+  - name: Weller WE1010NA
+    img: wellner
+    url: https://www.weller-tools.com/we1010na/
 
-- name: Reflow Oven T962
-  img: t962
-  url: https://www.amazon.com/dp/B01LZYEF90
+  - name: Eventek KPS305D DC Power Supply
+    img: eventek
+    url: https://www.amazon.com/dp/B071RNT1CD
 
-- name: Andonstar AD407 Digital Microscope
-  img: andonstar
-  url: https://www.amazon.com/dp/B07VK52X9C
+  - name: Reflow Oven T962
+    img: t962
+    url: https://www.amazon.com/dp/B01LZYEF90
 
+  - name: Andonstar AD407 Digital Microscope
+    img: andonstar
+    url: https://www.amazon.com/dp/B07VK52X9C
 
 Kniiting:
-- name: Brother 930E Knitting Machine
-  img: brother
-
-- name: Pfaff Creative 4.5 Embroidery Machine
-  img: pfaff
-  url: http://www.pfaff.com/en-CA/Machines/creative-4-5
+  - name: Brother 930E Knitting Machine
+    img: brother
 
+  - name: Pfaff Creative 4.5 Embroidery Machine
+    img: pfaff
+    url: http://www.pfaff.com/en-CA/Machines/creative-4-5
 
 Power Tools:
-- name: Hitachi C10FCG
-  img: hitachi
-  url: https://www.amazon.com/dp/B07217ZVP5
-
-- name: WEN 4208 Drill Press
-  img: drill-press
-  url: https://www.amazon.com/dp/B00HQONFVE
+  - name: Hitachi C10FCG
+    img: hitachi
+    url: https://www.amazon.com/dp/B07217ZVP5
 
-- name: WEN 3959 Band Saw
-  img: band-saw
-  url: https://www.amazon.com/dp/B077QMBTLP
+  - name: WEN 4208 Drill Press
+    img: drill-press
+    url: https://www.amazon.com/dp/B00HQONFVE
 
-- name: Black+Decker 20V Drill
-  img: black-decker
-  url: https://www.amazon.com/dp/B00C625KVE
+  - name: WEN 3959 Band Saw
+    img: band-saw
+    url: https://www.amazon.com/dp/B077QMBTLP
 
+  - name: Black+Decker 20V Drill
+    img: black-decker
+    url: https://www.amazon.com/dp/B00C625KVE
 
 AR/VR:
-- name: Vicon Motion Capture
-  img: vicon
-  url: https://www.vicon.com/
+  - name: Vicon Motion Capture
+    img: vicon
+    url: https://www.vicon.com/
 
```

Adding Meta Quest 2 (I'll need to check with Lora to see if she purchased any Quest 3's recently)
```suggestion

  - name: Meta Quest 2
    img: meta-quest-2
    url: https://www.meta.com/ca/quest2/
    
```